clwb/tests/integrationtests/com/google/idea/blaze/clwb/SimpleTest.java

```diff
@@ -44,4 +48,20 @@ private void checkTest() {
     assertContainsHeader("iostream", compilerSettings);
     assertContainsHeader("catch2/catch_test_macros.hpp", compilerSettings);
   }
+
+  private void checkXcode() throws IOException {
+    if (!SystemInfo.isMac) {
+      return;
+    }
+
+    final var compilerSettings = findFileCompilerSettings("main/test.cc");
+
+    final var compilerExecutable = compilerSettings.getCompilerExecutable();
+    assertThat(compilerExecutable).isNotNull();
+
+    final var scriptLines = Files.readAllLines(compilerSettings.getCompilerExecutable().toPath());
+
+    assertContainsPattern("export DEVELOPER_DIR=/.*/Xcode.*.app/Contents/Developer", scriptLines);
```

Comment:
I wonder why we do not have similar environment being injected for Apple Clang support in CLion.